internal/handlers/post_handler.go

```diff
@@ -165,45 +170,28 @@ func (handler *PostHandler) DeletePost(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Delete the hashtags associated with the post, if the hashtag isn't used also delete it
-	queryString = `DELETE FROM alpha_schema.many_posts_has_many_hashtags 
-       				WHERE post_id_posts = $1 RETURNING hashtag_id_hashtags`
-	rows, err := tx.Query(transactionCtx, queryString, postId)
-	if err != nil {
+	// Delete the post
+	queryString = "DELETE FROM alpha_schema.posts WHERE post_id = $1"
+	if _, err = tx.Exec(transactionCtx, queryString, postId); err != nil {
 		utils.WriteAndLogError(w, schemas.DatabaseError, http.StatusInternalServerError, err)
```

Comment:
Ich kann gerade in Postman nicht überprüfen, aber hatten wir nicht noch den Fehlerfall "Post not Found"?

Reply:
Überprüf ich in Zeile 152